browser/mapping.go

```diff
@@ -811,8 +812,34 @@ func mapWorker(vu moduleVU, w *common.Worker) mapping {
 func mapBrowserContext(vu moduleVU, bc *common.BrowserContext) mapping { //nolint:funlen
 	rt := vu.Runtime()
 	return mapping{
-		"addCookies":       bc.AddCookies,
-		"addInitScript":    bc.AddInitScript,
+		"addCookies": bc.AddCookies,
+		"addInitScript": func(script goja.Value) error {
+			if !gojaValueExists(script) {
+				return nil
```

Comment:
Previously this was running `AddInitScript` even without a Goja value. Just to let you know.

Reply:
I ran a test already with an empty script, and it is as good as returning `nil` and not doing anything.